pkg/controllers/management/feature/feature_handler.go

```diff
@@ -96,7 +96,7 @@ func (h *handler) syncHarvesterFeature(obj *v3.Feature) error {
 
 func (h *handler) toggleHarvesterNodeDriver(harvester string) error {
 	if val := features.GetFeatureByName(harvester).Enabled(); val {
-		m, err := h.nodeDriverController.Cache().Get(harvester)
+		m, err := h.nodeDriverController.Get(harvester, metav1.GetOptions{})
```

Comment:
This is not permanent. I'm not currently able to get the cache to work, so for now I need to change this to a full GET. DO NOT MERGE

Reply:
[This](https://github.com/rancher/rancher/pull/44766#discussion_r1565393800) should fix the problem with the cache.

Reply:
That did the trick!